code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,30 @@
+## CNCF सामुदायिक आचार संहिता v1.0
+
+### आचार संहिता का योगदानकर्ता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो रिपोर्टिंग issues के माध्यम से योगदान करते हैं, फीचर अनुरोध पोस्ट करते हैं, प्रलेखन अपडेट करते हैं, pull requset या patches, और अन्य गतिविधियों में भाग लेते हैं।
```

Comment:
"issue" can be written in the form of मसला or मुद्दा

Reply:
I know but when we use `मसला` word with `रिपोर्टिंग`, it doesn't match up.